packages/ethers5/src/client.ts

```diff
@@ -485,10 +480,10 @@ export class Web3Modal extends Web3ModalScaffold {
     const currentActiveWallet = window?.localStorage.getItem(EthersConstantsUtil.WALLET_ID)
     if (currentActiveWallet) {
       const currentProvider = this.EIP6963Providers.find(
-        provider => provider.name === currentActiveWallet
+        provider => provider.info.name === currentActiveWallet
```

Comment:
is this always the case? is there a case where info is not serialized? eg. non-6963 wallets?